assets/js/unregister-block-style.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * Unregister "Wide" Separator Style
+ */
+wp.domReady( function() => {
```

Comment:
This will need to use old-style syntax, we still need to be IE-compatible.

Reply:
👍 2a1317f